projects/demo-integrations/cypress/tests/kit/date/date-basic.cy.ts

```diff
@@ -18,6 +18,11 @@ describe('Date', () => {
                 ['12', '12', '12'.length],
                 ['121', '12.1', '12.1'.length],
                 ['1211', '12.11', '12.11'.length],
+                ['0', '0', 1],
+                ['00', '01', '01'.length],
+                ['000', '01.0', '01.0'.length],
+                ['0000', '01.01', '01.01'.length],
+                ['00000000', '01.01.0001', '01.01.0001'.length],
             ] as const;
 
```

Comment:
`2000/0|` => Type `0` => `2000/01|`

https://user-images.githubusercontent.com/35179038/225312813-99e8c063-b84f-4fc4-9245-30c5002128e6.MP4



Should it prevent typing for this case (only for basic typing, for paste/drop etc it okey now) ?

For example, the similar case:
```
`2000/1|` => Type 9 => no changes.
```

Reply:
@waterplea @splincode guys, what do you think ?